packages/vite-plugin-cloudflare/src/runner/module-runner.ts

```diff
@@ -5,11 +5,6 @@ import type { WrapperEnv } from './env';
 
 let moduleRunner: ModuleRunner;
 
-// TODO: node modules using process.env don't find `process` in the global scope for some reason
-//       for now we just create a `process` in the global scope but a proper solution needs to be
-//       implemented (see: https://github.com/flarelabs-net/vite-plugin-cloudflare/issues/22)
-(globalThis as Record<string, unknown>).process = { env: {} };
-
```

Comment:
I moved this polyfill to the integration test that needs it instead of turning it on for everyone.

I tried turning on nodejs_compat for that `react` library but then it broke in other ways...